packages/blocks/src/root-block/edgeless/components/auto-complete/auto-complete-panel.ts

```diff
@@ -512,9 +539,14 @@ export class EdgelessAutoCompletePanel extends WithDisposable(LitElement) {
         ? this.currentSource
         : this.edgeless.service.editPropsStore.getLastProps('shape');
 
-    const computedStyle = getComputedStyle(this.edgeless);
-    const stroke = computedStyle.getPropertyValue(strokeColor);
-    const fill = computedStyle.getPropertyValue(fillColor);
+    const stroke = this.edgeless.surface.themeObserver.getRealColor(
+      strokeColor,
+      DEFAULT_SHAPE_STROKE_COLOR
+    );
+    const fill = this.edgeless.surface.themeObserver.getRealColor(
```

Comment:
Same here.

Reply:
As you said before, a global themeObserver singleton would be more reasonable